extensions/sticky/locale/en.yml

```
@@ -6,6 +6,9 @@ flarum-sticky:
 
   # Translations in this namespace are used by the admin interface.
   admin:
+    settings:
+      filter_read_from_stickied_label: Filter read stickied (All Discussions)
```

I think it would be more understandable to say

```suggestion
      filter_read_from_stickied_label: Only sticky unread discussions
```

what do you think?

---

Much better! I'll update the label and variables so they make sense. Thanks :)